migrations.lua

```diff
@@ -123,5 +132,13 @@ return {
             { 'updated_at', types.time({ timezone = true }) },
             { 'offense_count', types.integer, { default = 1 } }
         })
+    end,
+
+    ['2020-10-22:0'] = function ()
+        schema.add_column('users', 'unique_email', types.text({ null = true, unique = true }))
+        -- We use an index on *non-unique* emails to be able to search related accounts.
+        -- We will rarely query by unique_email and thus no index is necessary.
+        schema.create_index('users', 'email', { unique = false })
+        update_user_views()
```

Comment:
This is a downside of the views approach, though it's relatively easy to notice if you forget this step.